rules/assertion-message.js

```diff
@@ -67,6 +67,9 @@ const schema = [{
 module.exports = {
 	create,
 	meta: {
+		docs: {
+			url: 'https://github.com/avajs/eslint-plugin-ava/tree/master/docs/rules/assertion-message.md'
```

Comment:
This should be  https://github.com/avajs/eslint-plugin-ava/blob/4211212daf1bfcfff3ebc5d4efdc4ba1a87acbf1/docs/rules/assertion-message.md. The rule was removed after 3.1.1 and the associated file does not exist in master anymore.

Reply:
Ah! A great example of why having the URL be unique to the rule is a good thing.